worker_pool/worker_pool.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"sync"
+	"time"
+)
+
+func main() {
+	var urls = []string{"https://www.google.com", "https://www.methods.co.uk", "https://www.github.com", "https://www.stackoverflow.com", "https://go.dev", "https://www.youtube.com", "https://www.ons.gov.uk", "https://coronavirusresources.phe.gov.uk/", "https://campaignresources.phe.gov.uk/resources", "https://www.twitter.com", "https://www.facebook.com"}
```

Comment:
not sure what the conventional max line length is for go, but I'd say these lines need to wrap at some point. 120 characters?

Reply:
can you vertically stack a statement like this?